include/hexahedron_element.tcc

```diff
@@ -518,4 +518,44 @@ inline Eigen::MatrixXi
   //clang-format on
   return indices;
 }
+
+
+//!            Behind: F4
+//!        3      F2       2
+//!          0_ _ _ _ _ _0
+//!         /|           /|
+//!        / |          / |
+//!     7 0_ |_ _ _ _ _0 6|
+//!       |  |         |  |
+//!    F3 |  |         |  |   F1
+//!       |  0_ _ _ _ _|_ 0
+//!       | / 0        | / 1
+//!       |/     F0    |/
+//!       0_ _ _ _ _ _ 0
+//!     4               5  
+//!         Front: F5 
+//!
+//!
+//! Bottom face: F0, Right face: F1, Top face: F2, 
+//! Left face: F3, Behind face: F4, Front face: F5
+//! Return indices of a face of the element
+template <unsigned Tdim, unsigned Tnfunctions>
+inline Eigen::VectorXi
```

Comment:
Return `Eigen::Matrix<int, 4, 1>` `int` or `unsigned`?

Reply:
I think the best is to use `unsigned`. Because of previous design using Eigen's typedef like `VectorXi` or `MatrixXi`, I used `int`. I have reverted back to `unsigned`